src/modules/rust.rs

```diff
@@ -99,14 +101,22 @@ fn env_rustup_toolchain(context: &Context) -> Option<String> {
     Some(val.trim().to_owned())
 }
 
-fn execute_rustup_override_list(cwd: &Path) -> Option<String> {
-    let Output { stdout, .. } = create_command("rustup")
-        .ok()?
-        .args(&["override", "list"])
-        .output()
-        .ok()?;
-    let stdout = String::from_utf8(stdout).ok()?;
-    extract_toolchain_from_rustup_override_list(&stdout, cwd)
+fn execute_rustup_override_list(context: &Context) -> Option<String> {
+    extract_toolchain_from_rustup_override_list(
+        &context.exec_cmd("rustup", &["default"])?.stdout,
```

Comment:
Should this not be

```suggestion
        &context.exec_cmd("rustup", &["override", "list"])?.stdout,
```

Reply:
thanks for the reminder, copy/paste errors still happen, no matter the experience :)

Reply:
fixed it, in the same commit. 

Or should I squash?